src/main/java/org/jabref/gui/externalfiles/DownloadExternalFile.java

```diff
@@ -310,8 +310,7 @@ private void downloadFinished() {
 
     private String getSuggestedFileName(String suffix) {
         String plannedName = FileUtil.createFileNameFromPattern(databaseContext.getDatabase(), entry,
-                Globals.prefs.get(JabRefPreferences.IMPORT_FILENAMEPATTERN),
-                Globals.prefs.getLayoutFormatterPreferences(Globals.journalAbbreviationLoader));
```

Comment:
Can you clarify some more why the `LayoutFormatterPreferences` are gone here? Doesn't this break existing formatting?

Reply:
See my e-mail reply below.